[2023-08-15 20:27:33,389] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-09T00:00:00+00:00 [queued]>
[2023-08-15 20:27:33,396] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-09T00:00:00+00:00 [queued]>
[2023-08-15 20:27:33,396] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:27:33,396] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-15 20:27:33,396] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:27:33,418] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta_semana> on 2023-01-09 00:00:00+00:00
[2023-08-15 20:27:33,421] {standard_task_runner.py:52} INFO - Started process 50718 to run task
[2023-08-15 20:27:33,426] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta_semana', 'scheduled__2023-01-09T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/dados_climatics.py', '--cfg-path', '/tmp/tmpuxo4eamv', '--error-file', '/tmp/tmpxnvhryxr']
[2023-08-15 20:27:33,426] {standard_task_runner.py:80} INFO - Job 136: Subtask cria_pasta_semana
[2023-08-15 20:27:33,509] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-09T00:00:00+00:00 [running]> on host ubuntu.internal.cloudapp.net
[2023-08-15 20:27:33,585] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta_semana
AIRFLOW_CTX_EXECUTION_DATE=2023-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-09T00:00:00+00:00
[2023-08-15 20:27:33,587] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-15 20:27:33,587] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/guilherme/airflow/semana=2023-01/2023-01-16"']
[2023-08-15 20:27:33,595] {subprocess.py:85} INFO - Output:
[2023-08-15 20:27:33,598] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-15 20:27:33,630] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta_semana, execution_date=20230109T000000, start_date=20230815T202733, end_date=20230815T202733
[2023-08-15 20:27:33,678] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-15 20:27:33,695] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-15 20:39:16,747] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-09T00:00:00+00:00 [queued]>
[2023-08-15 20:39:16,753] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-09T00:00:00+00:00 [queued]>
[2023-08-15 20:39:16,753] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:39:16,753] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-15 20:39:16,753] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:39:16,817] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta_semana> on 2023-01-09 00:00:00+00:00
[2023-08-15 20:39:16,821] {standard_task_runner.py:52} INFO - Started process 55708 to run task
[2023-08-15 20:39:16,826] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta_semana', 'scheduled__2023-01-09T00:00:00+00:00', '--job-id', '237', '--raw', '--subdir', 'DAGS_FOLDER/dados_climatics.py', '--cfg-path', '/tmp/tmppmd7f8rf', '--error-file', '/tmp/tmpwpq5pxwx']
[2023-08-15 20:39:16,827] {standard_task_runner.py:80} INFO - Job 237: Subtask cria_pasta_semana
[2023-08-15 20:39:16,908] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-09T00:00:00+00:00 [running]> on host ubuntu.internal.cloudapp.net
[2023-08-15 20:39:16,982] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta_semana
AIRFLOW_CTX_EXECUTION_DATE=2023-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-09T00:00:00+00:00
[2023-08-15 20:39:16,983] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-15 20:39:16,983] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/guilherme/airflow/dados/semana=2023-01/2023-01-16"']
[2023-08-15 20:39:16,991] {subprocess.py:85} INFO - Output:
[2023-08-15 20:39:16,994] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-15 20:39:17,025] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta_semana, execution_date=20230109T000000, start_date=20230815T203916, end_date=20230815T203917
[2023-08-15 20:39:17,078] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-15 20:39:17,097] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
